content/reference-transaction-format.md

```diff
@@ -724,6 +725,125 @@ In the above example with a ¥95/$15 offer and a ¥5/$2 offer, the situation is
 
 
 
+## PaymentChannelClaim
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/PayChan.cpp "Source")
+
+_Requires the [PayChan Amendment](concept-amendments.html#paychan)._
+
+Claim XRP from a payment channel, adjust its expiration, or both. This transaction can be used differently depending on the transaction sender's role in the specified channel:
+
+The **source address** of a channel can:
+
+- Send XRP from the channel to the destination with _or without_ a signed Claim.
+- Set the channel to expire as soon as the channel's `SettleDelay` has passed.
+- Clear a pending `Expiration` time.
+- Close a channel immediately if it has no XRP remaining in it.
```

Comment:
It means at the time the transaction is processed. (Normally you have to wait out a delay.)

How about this?

> Close a channel immediately. You can only do this if it has no XRP remaining in it.